.github/workflows/test.yml

```diff
@@ -15,22 +15,56 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.5', '3.8' ]
+        python-version: [ '3.8' ]
         stage: [ 'build', 'rc', 'production']
       fail-fast: false
     steps:
-    - uses: actions/checkout@v2
-    - name: Install libudunits2-dev
+    - uses: actions/checkout@v3
+    - name: additional required packages
       run: |
-        sudo apt install libudunits2-dev
+        sudo apt-get update &&
+        sudo apt-get install -y --no-install-recommends \
+        libudunits2-dev
+    - name: cartopy - install additional required packages
+      run: |
+        sudo apt-get install -y --no-install-recommends \
+        libproj-dev \
+        libgeos-dev \
+        libffi-dev \
+        cmake \
+        libgtest-dev \
+        build-essential \
+        curl \
+        libcurl4-openssl-dev
+    - name: cartopy - build Gtest needed for proj build
```

Comment:
I think we should use a binary if possible for this so the actions don't take ages